models/Document.php

```diff
@@ -42,6 +42,9 @@
  */
 class Document extends Element\AbstractElement
 {
+    private const EXCLUDED_TYPES = [
+        'folder'
```

Comment:
What's the reason behind this?
I see that may be inconsistent, since `Asset::getTypes()` and `DataObject::getTypes()` still return `folder`

Reply:
Sorry this was my mistake, I looked at the wrong config, "valid_tables" instead of the types and thought it has to be exluded.
I removed the line and updated the method